.github/workflows/update-docs.yml

```diff
@@ -40,43 +40,33 @@ jobs:
         with:
           repository: tauri-apps/tauri-search-bot
           path: tauri-search-bot
-      - name: install webkit2gtk
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y libgtk-3-dev webkit2gtk-4.0 libappindicator3-dev librsvg2-dev patchelf
 
-      # Rust
-      - name: generate rust docs
-        working-directory: ./tauri/core/tauri
-        run: cargo doc --no-deps
-      - name: run rustdocusaurus
-        uses: tauri-apps/rustdocusaurus/github-action@v1
-        with:
-          originPath: ./tauri/target/doc/
-          targetPath: ./tauri-docs/docs/en/api/rust/
-          sidebarPath: "${{ github.workspace }}/tauri-docs/sidebars/rustdoc.json"
-          linksRoot: ""
-          cratesToProcess: "tauri"
+      # Any Rust documentation is currently disabled while we're falling back to docs.rs
+
+      # - name: install webkit2gtk
+      #   run: |
+      #     sudo apt-get update
+      #     sudo apt-get install -y libgtk-3-dev webkit2gtk-4.0 libappindicator3-dev librsvg2-dev patchelf
 
-      # TypeScript
-      - name: install API deps
+      # # Rust
+      # - name: generate rust docs
+      #   working-directory: ./tauri/core/tauri
+      #   run: cargo doc --no-deps
+      # - name: run rustdocusaurus
+      #   uses: tauri-apps/rustdocusaurus/github-action@v1
+      #   with:
+      #     originPath: ./tauri/target/doc/
+      #     targetPath: ./tauri-docs/docs/en/api/rust/
+      #     sidebarPath: "${{ github.workspace }}/tauri-docs/sidebars/rustdoc.json"
+      #     linksRoot: ""
+      #     cratesToProcess: "tauri"
+
+      - name: Generate JS docs
         working-directory: ./tauri/tooling/api
-        run: yarn && yarn add typescript@4.2
-      - name: run typedocusaurus
-        uses: tauri-apps/typedocusaurus@v1
-        with:
-          originPath: ./tauri/tooling/api/
-          sidebarFile: "${{ github.workspace }}/tauri-docs/sidebars/typedoc.json"
-          targetPath: ${{ github.workspace }}/tauri-docs/docs/en/api/js
-          docusaurusPath: ./tauri-docs/
+        run: yarn && yarn generate-docs
 
-      # Moving docs for Indexation
-      - name: copy docs
-        working-directory: ./tauri
-        run: |
-          mv docs/sidebar.json ${{ github.workspace }}/tauri-docs/sidebars/core.json
-          cp -r docs/!(.templates) ${{ github.workspace }}/tauri-docs/docs/en
-          mv ARCHITECTURE.md ${{ github.workspace }}/tauri-docs/docs/en/about/architecture.md
```

Comment:
I'd love to keep this too

Reply:
https://github.com/tauri-apps/tauri/pull/3249#issuecomment-1016522707 Commented on this here, we can do whatever makes the most sense for this specific piece, though